pkg/relayer/config/types.go

```diff
@@ -2,10 +2,10 @@ package config
 
 import "net/url"
 
-type ProxyType int
+type ServerType int
```

Comment:
Is this still necessary?

Reply:
https://github.com/pokt-network/poktroll/pull/477#discussion_r1563826905